discover_overlay/discover_overlay.py

```diff
@@ -521,39 +562,21 @@ def entrypoint():
     else:
         logging.basicConfig(format=log_format)
     log.info("Starting Discover Overlay: %s",
-             pkg_resources.get_distribution('discover_overlay').version)
+         pkg_resources.get_distribution('discover_overlay').version)
 
     # Hedge against the bet gamescope ships with some WAYLAND_DISPLAY
     # Compatibility and we're not ready yet
     if 'GAMESCOPE_WAYLAND_DISPLAY' in os.environ:
         os.unsetenv("WAYLAND_DISPLAY")
-
-    # Catch any errors and log them
+    # errors
     try:
-        if "--rpc" in sys.argv:
-            # Send command to overlay
-            line = ""
-            for arg in sys.argv[1:]:
-                line = f"{line} {arg}"
-            with open(rpc_file, "w", encoding="utf-8") as tfile:
-                tfile.write(line)
-                log.warning("Sent RPC command")
-        else:
-            if "-c" in sys.argv or "--configure" in sys.argv:
-                # Show config window
-                _settings = MainSettingsWindow(
-                    config_file, rpc_file, channel_file, sys.argv[1:])
-                Gtk.main()
-            else:
-                # Tell any other running overlay to close
-                with open(rpc_file, "w", encoding="utf-8") as tfile:
-                    tfile.write("--close")
-                # Show the overlay
-                Discover(rpc_file, config_file, channel_file,
-                         debug_file, sys.argv[1:])
-        return
+        with open(rpc_file, "w", encoding="utf-8") as tfile:
```

Comment:
Removing RPC entirely means each new command kills the last and has to establish a connection with the local Discord client before anything can be actioned. Also it will cause a visible loss of overlay for a fraction of a second which I absolutely will find jarring just for a mute/unmute